packages/gdl-frontend/components/Layout/index.js

```diff
@@ -67,7 +139,81 @@ class Layout extends React.Component<Props, State> {
           onClose={() => this.setState({ drawerIsOpen: false })}
           isOpen={this.state.drawerIsOpen}
         />
-        {wrapWithMain ? <Main component="main">{children}</Main> : children}
+        {wrapWithMain ? (
+          <Main square component="main">
+            {children}
+          </Main>
+        ) : (
+          children
+        )}
+        <Container size="large" stickToEdgeOnLargeScreens width="100%">
+          <Footer>
+            <div
+              css={[
+                {
+                  order: 2,
+                  border: 'solid 1px #e3e3e3',
+                  width: '100%',
+                  margin: '10px'
+                },
+                media.tablet({ display: 'none' })
+              ]}
+            />
+
+            <LinkGrid>
```

Comment:
Jeg lurer på om vi kanskje skal kjøre med 2 kolonner med linker i stedet for 3 på mobil